src/webview/MessageList.js

```diff
@@ -149,7 +149,7 @@ class MessageList extends Component<Props> {
   webview: ?WebView;
   readyRetryInterval: IntervalID | void;
   sendUpdateEventsIsReady: boolean;
-  unsentUpdateEvents: WebViewUpdateEvent[] = [];
+  unsentUpdateEvents: WebViewInboundEvent[] = [];
```

Comment:
Grepping for `pdateEvent` (or `updateevent` case-insensitive), there's still a few identifiers here and also in js.js which use that terminology. Those would be good to update too, to match. Maybe cleanest to do in a second commit right after this one.